lib/repositories/forms_repository.rb

```diff
@@ -16,6 +16,14 @@ def get_by_org(org)
   end
 
   def update(form)
+    if form[:live_at]
+      the_form = @database[:forms].where(id: form[:form_id])
+      raise "Invalid email" unless form[:submission_email].include?("@")
+      raise "Form has no name" unless form[:name]
+      raise "Form has no pages" unless @database[:pages].where(form_id: form[:form_id]).count > 0
+      raise "Form has a privacy policy" unless form[:privacy_policy_url]
```

Comment:
```suggestion
      raise "Form has no privacy policy URL" unless form[:privacy_policy_url]
```

Reply:
Thanks so much Al, but maybe we'll have to throw (ha) away this exception system, which is what I've been noodling with? Lets talk about it